coffee-chats/src/main/java/com/google/step/coffee/entity/ChatRequestBuilder.java

```diff
@@ -14,21 +13,23 @@
   private List<Date> dates = new ArrayList<>();
   private int minPeople = 1;
   private int maxPeople = 4;
-  private int duration = 30;
+  private int durationMins = 30;
```

Comment:
Why not Duration?

Reply:
It was pointed out earlier that it was not very clear from the code/documentation other than a comment in the builder, what the units of the duration were.

Reply:
When converting to Duration I missed the ChatRequest class and builder. Fixed.